libs/ledger-live-common/src/families/aptos/logic.test.ts

```diff
@@ -81,31 +81,25 @@ describe("Aptos logic ", () => {
 
   describe("normalizeTransactionOptions", () => {
     it("should normalize transaction options", () => {
-      const options: Transaction["options"] = {
+      const options: Transaction["estimate"] = {
```

Comment:
are you saying something like

```js
const options: TransactionEstimate = {
```

you're setting const `options` to be of type `x`, it is weird to do as `Transaction["estimate"]` becomes unclear what it should be it's type

Reply:
also, call it `estimate` or `TransactionOptions` or the other way around